README.md

```diff
@@ -78,26 +78,28 @@ You can use ion-js either as a Node.js module or inside an html page.
     ```javascript
     var ionJs = require("ion-js")
 
-    var ionData = "{ hello: \"Ion\" }"; 
-    var ionReader = ionJs.makeReader(ionData); 
-    ionReader.next(); 
-    ionReader.stepIn(); 
-    ionReader.next(); 
-    var hello = ionReader.fieldName(); 
-    var ion = ionReader.stringValue(); 
-    ionReader.stepOut(); 
+    var ionData = "{ hello: \"Ion\" }";
+    var ionReader = ionJs.makeReader(ionData);
+    ionReader.next();
+    ionReader.stepIn();
+    ionReader.next();
+    var hello = ionReader.fieldName();
+    var ion = ionReader.stringValue();
+    ionReader.stepOut();
     console.log(ion.concat(", ").concat(hello));
     ```
-    [Try it yourself](https://npm.runkit.com/ion-js). 
+
+    **Warning:** RunKit has an old version of ion-js and the APIs have change considerably since them. Please
```

Comment:
Couple of suggested edits:

1. > the APIs have change**d** considerably

2. > ~~Please install it locally using NPM to try it out, [issue](https://github.com/amzn/ion-js/issues/201) tracking this problem~~
    > Please [install `ion-js` locally using NPM](https://github.com/amzn/ion-js/#npm) to try it out. For status updates, refer to [this issue](https://github.com/amzn/ion-js/issues/201).